packages/components/src/components/horizontal-swipe/horizontal-swipe.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Swipeable } from 'react-swipeable';
+
+const HorizontalSwipe = ({
```

Comment:
is it possible to use the existing `SwipeableWrapper` component instead of creating a new one?

Reply:
I tried using that but there were limitations to how swift the swipe was and blocking vertical swiping completely.